### DISCUSSION TITLE
Job Requirements Kekinian

### DISCUSSION THREAD

**Discussion by Falahyan-104919** [deaafrizal/forum#9] [The Lounge]
hello forum! izin memperkenalkan diri saya mahasiswa tingkat akhir yang dalam hitungan bulan mau terjun ke dunia sebenarnya, wish me luck yaa!🙌🏻

kembali ke title discussion yang mau di bahas di forum diskusi ini. Realita/kondisi di lapangan tuh emangnya sesuai ya sama skill requirements yang mereka set untuk kandidat di posisi itu? boleh dong sharing pengalaman kalian pas masih apply cari-cari kerjaan. seberapa akurat skill requirements sama actual workload yang dikasih?

**Comment by teguhkrniawan** [comment]
Sepengalaman ku melamar kerja di perusahaan yang enggak punya nama gede ketika requirement diminta berbagai macam bahasa pemrograman, aslinya yang dipakai cuman 2 bahasa. Nanti ketika di interview biasanya user akan menanyakan spesifik bahasa pemrogramannya, atau pas test code juga akan ketauan mereka prefer ke bahasa apa. Untuk version control itu wajib ya, kalau untuk kebutuhan docker, CI/CD, semacemnya kebetulan akunya belum punya pengalaman sampai disana, dan tempat kerjaku dulu juga belum menerapkan hal tersebut.

misal "mas nya bisa bahasa JS ? pernah pakai framework apa ? pernah makai db postgree gak ? dan blah blah"

nah kalau aku sih, asumsinya itu adalah tech stack yang digunakan, dan bener aja itu memang tech stack mereka.

**Reply by Falahyan-104919** [reply]
jadi bisa dibilang itu skill requirements sifatnya global ya bang, nanti bakal di adjust lagi sama kandidatnya gitu? atau gimana bang. apa emang mereka nyari kandidat ideal nya harus memenuhi semua kriteria yang mereka mau?

**Reply by teguhkrniawan** [reply]
menurutku pasti mereka cari kandidat yang ideal dengan mereka, minimal punya peluang untuk bertumbuh (yang kalau dikasi insight sat set sat set paham) gitu dan tentu dengan attitude yang baik pula.

**Reply by Falahyan-104919** [reply]
oooh gitu bang okee paham saya sekarang, makasih yaa bang insight nya🙌🏻
